test/pathBuilder.test.js

```diff
@@ -71,15 +71,15 @@ describe('documentation paths', () => {
     });
 
     it('css distribution directory path should be correct', () => {
-        expect(pathBuilder.docsCssDistDir).toBe('./docs/dist/assets/css');
+        expect(pathBuilder.docsCssDistDir).toBe('./docs/dist/assets/dist/css');
```

Comment:
I did originally remove the additional `dist` directory, however the issue is that the SCSS holds the path as a variable which the background image `url` properties.